Test/Integration/ConfigTest.php

```diff
@@ -218,15 +218,9 @@ public function testExtraSettings()
         $sections = ['products', 'categories', 'pages', 'suggestions'];
 
         foreach ($sections as $section) {
+            $this->algoliaHelper->waitLastTask();
```

Comment:
On the first iteration of this loop it seems to me that we would call two wait operations in [immediate succession](https://github.com/algolia/algoliasearch-magento-2/pull/1620/files#diff-e50009685b9036004740bf0f93d4afd3bff470989f1fd0f93a7886fa0bb429e0R216)? Should this perhaps be located towards the end of the loop?

Reply:
Fixed